### DISCUSSION TITLE
Creating a View or Viewport Layer

### DISCUSSION THREAD

**Discussion by chrisgervang** [visgl/deck.gl#6872] [Ideas]
While learning to use all of the different views it would be really useful to visualize frustum and interact with all of the various view objects in the scene.

The camera objects on threejs.org/editor is a great example of what this could look like.

A layer like this could be used to improve our documentation on each view page with an interactive demo. It could also be useful to have this layer when designing new types of views, and learning the unique geospatial features implemented in our views.

I want to build this for View objects so that they can be easily interchanged with active views, but I'm curious if there are reasons this needs to be implemented with Viewport objects instead.

**Comment by ibgreen** [comment]
As a possible source of inspiration, the I3S team in loaders.gl have added [a minimap](https://loaders.gl/examples/i3s-debug) showing the view frustum of the of main map among other things.

<img width="1492" alt="Screen Shot 2022-05-02 at 8 46 47 AM" src="https://user-images.githubusercontent.com/7025232/166264456-e9ed113e-0efa-4172-8f83-7927a3a1f694.png">
